### ISSUE TITLE
Undefined property error in Laravel 5.3

### ISSUE THREAD

**Issue by petemolinero** [phaza/laravel-blade-inline-svg#1]
Hi,

I'm using your package with Laravel 5.3.

When I try to use the `svg` directive like so:

```
@svg(mySvg.svg)
```

I get the following error:

```
ErrorException in 4c4c59e2c1e7b3f1bf26fd4fe584c3981f89fc8a.php line 54:
Undefined property: Phaza\InlineSvg\Implementations\SvgInliner::$rendermySvg (View: G:\myproject\resources\views\layouts\home.blade.php) (View: G:\myprojectl\resources\views\layouts\home.blade.php)
```

Any thoughts?

Pete

**Comment by phaza** [comment]
I would do a search in your project for "rendermysvg", as I have no idea
where that property name comes from. It surely isn't in my code.
If you can't figure it out, I need to see your actual views.

Let me know what you find.
ons. 26. okt. 2016 kl. 23.39 skrev Pete Molinero notifications@github.com:

**Comment by petemolinero** [comment]
Hi @phaza ,

Thanks for the reply. `rendermySvg` isn't in the project.

My only @svg call anywhere in a view is:

```
@svg(mySvg.svg)
```

The symptom of the error is that it's actually taking this line:

```
// Line 12 of BladeInlinerSvgServiceProvider.php
return "<?php echo app('svginliner')->render{$args}; ?>";
```

and viewing the `->render{$args};` parts as `->$rendermySvg` (mySvg being the value--minus the .svg--of the argument I'm passing in. So it's concatenating the filename onto the render method and treating the resultant string as a variable to use as a property name. Pretty wacky.

Any ideas at all?

**Comment by phaza** [comment]
I suspect something has changed in the way Laravel treats blade directives.
When I made this, the parentheses would be part of arguments passed to the
callback, hence the slightly awkward render call.

You could try @svg(("mySvg.svg")), although it is a hack. Best would
probably be to fix the real culprit here :)
path needs to be a string at least, or the call will fail.

**Comment by petemolinero** [comment]
@phaza Yeah, I had thought that the path needed to be a string, but then I noticed in your test you had written:

```
// From tests/testview.blade.php
@svg(./tests/laravel.svg)
```

And anyways using a string had produced the error: 

```
Parse error: syntax error, unexpected '"mySvg.svg"' (T_CONSTANT_ENCAPSED_STRING)
```

BUT your suggestion of `@svg(("mySvg.svg"))` worked! I thought I'd read somewhere that 5.3 still used the parenthesis, but looks like I was mistaken!

**Comment by optimys** [comment]
Hi, @phaza Will you fix that issue for Laravel 5.3. 
Not digging too much but if i change ->render{$args}; to ->render**(**$args**)**; it's works for me.
